File: app/src/main/java/com/ianfield/bodyscoring/widgets/DividerColorItemDecoration.java
package com.ianfield.bodyscoring.widgets;

import android.content.Context;
import android.support.v4.content.ContextCompat;
import android.support.v7.widget.DividerItemDecoration;

import com.ianfield.bodyscoring.R;

/**
 * Created by ianfield on 20/12/2016.
 */

public class DividerColorItemDecoration extends DividerItemDecoration {

    public DividerColorItemDecoration(Context context, int orientation) {
        super(context, orientation);
        setDrawable(ContextCompat.getDrawable(context, R.drawable.line_divider));
    }
}

File: app/src/main/java/com/ianfield/bodyscoring/widgets/DividerColorItemDecoration.kt
package com.ianfield.bodyscoring.widgets

import android.content.Context
import androidx.core.content.ContextCompat
import androidx.recyclerview.widget.DividerItemDecoration

import com.ianfield.bodyscoring.R

/**
 * Created by ianfield on 20/12/2016.
 */

class DividerColorItemDecoration(context: Context, orientation: Int) : DividerItemDecoration(context, orientation) {

    init {
        setDrawable(ContextCompat.getDrawable(context, R.drawable.line_divider)!!)
    }
}